tests/SourceTest.php

```diff
@@ -2,211 +2,98 @@
-        $source->metadata['foo'] = 'bar';
-        $source->save();
-        $this->assertSame($source->metadata['foo'], 'bar');
+        $resource = Source::retrieve(TEST_RESOURCE_ID);
+        $resource->metadata["key"] = "value";
+        $this->expectsRequest(
+            'post',
+            '/v1/sources/' . TEST_RESOURCE_ID
+        );
+        $resource->save();
+        $this->assertSame("Stripe\\Source", get_class($resource));
     }
 
-    public function testSaveOwner()
+    public function testIsUpdatable()
     {
-        $response = array(
-            'id' => 'src_foo',
-            'object' => 'source',
-            'owner' => array(
-                'name' => null,
-                'address' => null,
-            ),
-        );
-        $this->mockRequest(
-            'GET',
-            '/v1/sources/src_foo',
-            array(),
-            $response
+        $this->expectsRequest(
+            'post',
+            '/v1/sources/' . TEST_RESOURCE_ID
         );
-
-        $response['owner'] = array(
-            'name' => "Stripey McStripe",
-            'address' => array(
-                'line1' => "Test Address",
-                'city' => "Test City",
-                'postal_code' => "12345",
-                'state' => "Test State",
-                'country' => "Test Country",
-            )
-        );
-        $this->mockRequest(
-            'POST',
-            '/v1/sources/src_foo',
-            array(
-                'owner' => array(
-                    'name' => "Stripey McStripe",
-                    'address' => array(
-                        'line1' => "Test Address",
-                        'city' => "Test City",
-                        'postal_code' => "12345",
-                        'state' => "Test State",
-                        'country' => "Test Country",
-                    ),
-                ),
-            ),
-            $response
-        );
-
-        $source = Source::retrieve('src_foo');
-        $source->owner['name'] = "Stripey McStripe";
-        $source->owner['address'] = array(
-            'line1' => "Test Address",
-            'city' => "Test City",
-            'postal_code' => "12345",
-            'state' => "Test State",
-            'country' => "Test Country",
-        );
-        $source->save();
-        $this->assertSame($source->owner['name'], "Stripey McStripe");
-        $this->assertSame($source->owner['address']['line1'], "Test Address");
-        $this->assertSame($source->owner['address']['city'], "Test City");
-        $this->assertSame($source->owner['address']['postal_code'], "12345");
-        $this->assertSame($source->owner['address']['state'], "Test State");
-        $this->assertSame($source->owner['address']['country'], "Test Country");
+        $resource = Source::update(TEST_RESOURCE_ID, array(
+            "metadata" => array("key" => "value"),
+        ));
+        $this->assertSame("Stripe\\Source", get_class($resource));
```

Comment:
@ob-stripe  Flagging that we don't test owner updated anymore and might want to.